### PR TITLE
Remove references in monitors to hqes* and hqproxy0 servers

### PR DESCRIPTION
**Pull request by nospame**
Removes references in monitors to old servers that had their DNS entries removed as part of https://dimagi.atlassian.net/browse/SAAS-15848

##### Environments Affected
None, I think

### CODE REVIEW

File: src/commcare_cloud/manage_commcare_cloud/monitors/cert_expire.yml [hunk 1]
```diff
@@ -17,6 +17,6 @@ options:
   silenced: {}
   thresholds: {critical: 5, ok: 1, warning: 1}
   timeout_h: 0
-query: '"http.ssl_cert".over("*").exclude("environment:64-test","host:hqproxy0").by("host","instance","url").last(6).count_by_status()'
+query: '"http.ssl_cert".over("*").exclude("environment:64-test").by("host","instance","url").last(6).count_by_status()'
```
Comment (millerdev) [inline src/commcare_cloud/manage_commcare_cloud/monitors/cert_expire.yml:20]: I think this change will conflict with https://github.com/dimagi/commcare-cloud/pull/6361, so whichever is merged second will need to be have merge conflicts resolved.

**Review by millerdev — COMMENTED** [review]
I assume this is fine, although I don't know what these were used for in the past. Maybe @dannyroberts should review?

**Review by dannyroberts — APPROVED** [review]
This is fine, but also everything under `src/commcare_cloud/manage_commcare_cloud/monitors/` is essentially unused at this point as far as I know

File: src/commcare_cloud/manage_commcare_cloud/monitors/cert_expire.yml (hunk 1 above)
Comment (nospame) [reply src/commcare_cloud/manage_commcare_cloud/monitors/cert_expire.yml]: You are correct, of course. https://github.com/dimagi/commcare-cloud/pull/6365/commits/06006d199d704547cf6b938ad6b9558d6ebb08df

**Review by millerdev — APPROVED** [review]

**Pull request merged**